projects/ngrx.io/tools/stackblitz-builder/builder.js

```diff
@@ -1,60 +1,47 @@
 'use strict';
 
 // Canonical path provides a consistent path (i.e. always forward slashes) across different OSes
-var path = require('canonical-path');
-var Q = require('q');
-var _ = require('lodash');
-var jsdom = require("jsdom");
-var fs = require("fs-extra");
-var globby = require('globby');
+const path = require('canonical-path');
+const jsdom = require('jsdom');
+const fs = require('fs-extra');
+const globby = require('globby');
 
-var regionExtractor = require('../transforms/examples-package/services/region-parser');
+const regionExtractor = require('../transforms/examples-package/services/region-parser');
 
 class StackblitzBuilder {
   constructor(basePath, destPath) {
     this.basePath = basePath;
     this.destPath = destPath;
 
-    // Extract npm package dependencies
-    var packageJson = require(path.join(__dirname, '../examples/shared/boilerplate/cli/package.json'));
-    this.examplePackageDependencies = packageJson.dependencies;
-
-    // Add unit test packages from devDependency for unit test examples
-    var devDependencies = packageJson.devDependencies;
-    this.examplePackageDependencies['jasmine-core'] = devDependencies['jasmine-core'];
-    this.examplePackageDependencies['jasmine-marbles'] = devDependencies['jasmine-marbles'];
-
     this.copyrights = {};
-
     this._buildCopyrightStrings();
   }
 
   build() {
     this._checkForOutdatedConfig();
```

Comment:
`_checkForOutdatedConfig` can also be removed because we don't have any plnkr examples.